.github/workflows/unit-test.yml

```diff
@@ -153,3 +154,16 @@ jobs:
         run: |
           cfn-lint --version
           cfn-lint -I -t ./deploy/cloudformation/elastic-agent-ec2.yml
+
+  terraform-linter:
+    name: terraform-lint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Init Hermit
+        run: ./bin/hermit env -r >> $GITHUB_ENV
+
+      - name: Terraform fmt
```

Comment:
Lets have it atm only on the weekly env dir

Reply:
I don't mind changing it, but why we wouldn't want it to run on the entire solution? 

The CI passes, and it will keep all the terraform in format.